.github/workflows/ci-weekly.yml

```diff
@@ -1,74 +1,164 @@
-name: CI - FAST Validation
+name: CI - FAST
 
 on:
-  schedule:
```

Comment:
I want to continue and run this only on cron-job once per week.  Please re-add schedule

Reply:
ok. I will change the to on schedule